netbox_librenms_plugin/templates/netbox_librenms_plugin/_interface_sync_content.html

```
@@ -4,22 +4,41 @@
+</div>
+{% endif %}
+
+{# Migrated donors must not sync: hiding only the button leaves the POST form live (Enter in a filter still submits), so drop the form in migrated mode. #}
 {% with model_name=interface_sync.object|meta:"model_name" %}
+{% if migrated_to_marker %}
+<div>
+{% else %}
 <form method="post"
     action="{% url 'plugins:netbox_librenms_plugin:sync_selected_interfaces' object_type=model_name object_id=interface_sync.object.pk %}?interface_name_field={{ interface_name_field }}">
-    {% endwith %}
+    {# Form-only inputs live inside the form branch so migrated (donor) mode, which renders a plain <div> wrapper, never emits hidden fields outside any <form>. #}
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </form> ], start tag match failed [ <form> ] on line 21.

(tag-pair)